app-crypt/acme-sh/acme-sh-2.7.9.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN/-/.}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="An ACME Shell script"
+HOMEPAGE="https://github.com/Neilpang/acme.sh"
+SRC_URI="https://github.com/Neilpang/${MY_PN}/archive/${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="apache2 +cron nginx libressl standalone"
+
+RDEPEND="net-misc/curl
+	|| ( net-analyzer/netcat6 net-analyzer/openbsd-netcat )
+	sys-apps/iproute2
+	sys-apps/net-tools
```

Comment:
and this

Reply:
```
ebuild-x64-acme /usr/share/acme.sh # grep netstat acme.sh 
  if _exists "netstat"; then
    _debug "Using: netstat"
    if netstat -h 2>&1 | grep "\-p proto" >/dev/null; then
      #for windows version netstat tool
      netstat -an -p tcp | grep "LISTENING" | grep ":$_port "
      if netstat -help 2>&1 | grep "\-p protocol" >/dev/null; then
        netstat -an -p tcp | grep LISTEN | grep ":$_port "
      elif netstat -help 2>&1 | grep -- '-P protocol' >/dev/null; then
        netstat -an -P tcp | grep "\.$_port " | grep "LISTEN"
      elif netstat -help 2>&1 | grep "\-p" >/dev/null; then
        netstat -ntpl | grep ":$_port "
        netstat -ntl 2>/dev/null | grep ":$_port "
```
```
ebuild-x64-acme /usr/share/acme.sh # which netstat
/bin/netstat
ebuild-x64-acme /usr/share/acme.sh # equery b /bin/netstat
 * Searching for /bin/netstat ... 
sys-apps/net-tools-1.60_p20161110235919 (/bin/netstat)
```

Reply:
these are packages that belong the @system set of gentoo, you should never put packages from this set as dependencies unless the specific reason

Reply:
sys-apps/net-tools removed.